docs/UserGuide.md

```diff
@@ -93,10 +94,30 @@ Important warnings that you should keep in mind while using this User Guide.
 ## Quick start
 <IMPROVE BY ADDING SCREENSHOTS AND LINKS>
 
-1. Ensure Java 11 is installed in your computer. If it's not, install it [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
-2. Download the latest ConnectCare.jar <Link here>
-3. Copy the jar file to the folder you want to use as the home folder
-4. Double-click the file to start the application!
+This section allows you to start caring for your clients as soon as possible!
+
+### Installation and Startup Instructions
+1. To use ConnectCare you will require Java 11 to be installed in your computer. 
+   1. If you do not have Java 11, install it [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
+   2. If you have Java installed, check that it is the correct version:
+   
+       Windows or macOS users might find [this guide](https://blog.hubspot.com/website/check-java-verison) useful.
+
+2. Download the latest release of `addressbook.jar` [here](https://github.com/AY2324S2-CS2103T-W12-4/tp/releases).
+
+<div class="image-container" align="middle" style="display:flex">
+    <pic src="images/quickstart/latest-release.png" alt="jar_file">
+        <markdown> Look out for the latest release </markdown>
+    </pic>
+    <pic src="images/quickstart/jar-file.png" alt="jar_file">
+        <markdown> The JAR file can be found at the bottom of the page </markdown>
+    </pic>
+</div>
+
+3. Copy the jar file to the folder you want to use as the home folder.
+4. Double-click the file to start the application! Alternatively, use [this guide](https://www.baeldung.com/java-run-jar-with-arguments) to run the `.jar` file and start the application.
```

Comment:
I think the last link is a bit too complicated, so we can remove it

Reply:
I changed it to this link instead: https://www.wikihow.com/Run-a-.Jar-Java-File
Is it better? Or should I remove the hyperlink altogether?

Reply:
I think maybe, replace "Alternatively, use [this guide] ..." with "If you are facing issues, you can consult \[this guide](https://www.wikihow.com/Run-a-.Jar-Java-File)."

Reply:
The link LGTM!